touchforms/formplayer/api.py

```diff
@@ -320,6 +322,10 @@ def post_data(data, auth=None, content_type="application/json"):
     if auth:
         d['hq_auth'] = auth.to_dict()
     # just default to old server for now
+
+    if toggles.SMS_USE_FORMPLAYER:
+        return formplayer_post_data_helper(d, auth, content_type, get_nimbus_url() + "/" + data["action"])
```

Comment:
`perform_experiment` calls `get_candidate_session_data` which sets the values of both `session_id` and `session-id` to the same value. is session id already being set to the right param here or do both need to be set here too?

Reply:
Great point... also occurs to me that when we switch this over we'll want to continue doing the lookups between `touchforms` and `formplayer` `session_id`s so that we don't break any ongoing sessions